docs/customization.rst

```diff
@@ -176,15 +176,27 @@ Configuration Options
 
     .. themeconf:: repo_url
 
-        Set the repo url for the link to appear.
+        The link to the repository will be rendered next to the search bar on big screens and as
```

Comment:
"large screens"

Reply:
+1 Admittedly, this was copied from upstream's docs.